lib/studies/reply/mail_card_preview.dart

```diff
@@ -26,16 +28,15 @@ class MailPreviewCard extends StatelessWidget {
   final Email email;
   final VoidCallback onDelete;
   final VoidCallback onStar;
+  final bool isStarred;
+  final bool onStarredMailbox;
 
   @override
   Widget build(BuildContext context) {
     final theme = Theme.of(context);
 
-    final currentEmailStarred = Provider.of<EmailStore>(
-      context,
-      listen: false,
-    ).isEmailStarred(email);
-
+    // TODO(shihaohong): State restoration of mail view page is
```

Comment:
Oh, did you by any chance take a look at OpenContainer to see what it would take to make that restorable?

Reply:
Discussion continued in https://github.com/flutter/flutter/issues/69924.